presto-main/src/test/java/io/prestosql/server/TestDynamicFilterService.java

```diff
@@ -296,12 +346,33 @@ public void testReplicatedDynamicFilter()
                 singleValue(INTEGER, 1L))));
         assertTrue(dynamicFilter.isComplete());
         assertEquals(dynamicFiltersStageSupplier.getRequestCount(), 1);
+        DynamicFiltersStats stats = dynamicFilterService.getDynamicFilteringStats(queryId, session);
+        assertEquals(stats.getNumTotalDynamicFilters(), 1);
+        assertEquals(stats.getNumDynamicFiltersCompleted(), 1);
+        assertEquals(stats.getNumReplicatedDynamicFilters(), 1);
+        assertEquals(stats.getNumRepartitionedDynamicFilters(), 0);
+        assertEquals(
+                stats.getDynamicFilterDomainStats(),
+                ImmutableList.of(new DynamicFilterDomainStats(filterId1, Domain.create(
```

Comment:
put `filterId1, Domain.create` args in newlines,
use `SortedRangeSet.of` instead